prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -527,6 +528,9 @@ ssize_t rxr_pkt_init_medium_msgrtm(struct rxr_ep *ep,
 	if (ret)
 		return ret;
 
```

Comment:
Could u refresh my memory why this mr registration is tried for medium msg transfer only?

Reply:
It is not only for medium, but also for long-cts. These two protocols use this function because they can use memory registration opportunistically. e.g. they can use MR but does not rely on MR. Other protocols is not like that. Eager does not use MR at all. Long read rely on MR. Will update the comment to make it more clear.

Reply:
Updated comment in new revision.